src/diagram/Field.test.ts

```diff
@@ -0,0 +1,21 @@
+import { Field } from "./Field";
+
+test('Field constructor', () => {
+    const f = new Field("test", 1);
+    expect(f.name).toBe("test");
+    expect(f.length).toBe(1);
+});
+
+test('Field setName', () => {
```

Comment:
change the name to `Field name setter`

Reply:
Updated, please refer to 
[c14362b](https://github.com/Jerrylum/protocol-diagram/pull/8/commits/c14362b46accd850aaa7de3765b4c67e42df8132)